keyboards/rico/phoenix_project_no1/info.json

```diff
@@ -0,0 +1,120 @@
+{
+    "usb": {
+        "vid": "0x21C0",
+        "pid": "0x9901",
+        "device_version": "1.0.0"
+    },
+    "keyboard_name": "Phoenix Project No 1",
+    "manufacturer": "Rico",
+    "url": "https://github.com/mymakercorner/Phoenix_Project_No1",
+    "maintainer": "Rico",
+    "processor": "RP2040",
+    "bootloader": "rp2040",
+    "board": "GENERIC_RP_RP2040",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": false,
+        "rgblight": true
+    },
+    "diode_direction": "COL2ROW",
+    "matrix_pins": {
+        "cols": ["GP24", "GP0", "GP14", "GP13", "GP12", "GP10", "GP9", "GP8", "GP7", "GP6", "GP5", "GP4", "GP3", "GP25", "GP2", "GP1"],
+        "rows": ["GP27", "GP29", "GP19", "GP20", "GP26"]
+    },
+    "rgblight": {
+        "led_count": 16,
+        "pin": "GP28",
```

Comment:
```suggestion
    "ws2812": {
        "pin": "GP28"
    },
    "rgblight": {
        "led_count": 16,
```

Reply:
Your comment in b705fd274fbfce41219be2d52c9dc9d39e315e50 is incorrect for current `master`. This must be **resovled** per the above suggestion for this PR to be considered mergable.

Reply:
The master branch had this feature added since I made my branch, sorry for the confusion ...
Merged master into my branch and applying your suggestion.